LEAF_Request_Portal/sources/Form.php

```diff
@@ -4675,31 +4721,71 @@ private function isIndicatorOrphan($indicator, &$indicatorList)
      */
     public function copyAttachment($indicatorID, $fileName, $recordID, $newRecordID, $series): array|int
     {
-        if (!is_numeric($recordID) || !is_numeric($indicatorID) || !is_numeric($series))
-        {
-            $errors = array('type' => 2);
-            return $errors;
+        if (!is_numeric($recordID) || !is_numeric($indicatorID) || !is_numeric($series)) {
+            http_response_code(400);
+            error_log("Copy attachment error: Invalid parameters");
+            return ['success' => false, 'message' => 'Invalid parameters', 'type' => 2];
```

Comment:
Avoid using arbitrary return structures such as this. The HTTP response code + return human readable error string would be ideal.

However in this case since this function relies on `error_get_last()`, which returns an array, our return should more closely align with that format.